alpm-utils/src/alpm.rs

```diff
@@ -1,22 +1,24 @@
-use crate::depends::satisfies_ver;
+//! Extension methods for the [`Alpm`] type.
 
-use alpm::{Alpm, Result, Package, Depend};
+use crate::DbListExt;
+use alpm::{Alpm, Package};
 
-pub trait AlpmExt {
-    fn find_local_satisfier<S: Into<String>>(&self, pkg: S) -> Result<Option<Package>>;
-}
+/// All official packages.
```

Comment:
There's no concept off "official" packages. Comment should describe the concept of packages that can also be found in a syncdb.